src/components/HighlightedCode/HighlightedCode.js

```diff
@@ -35,8 +35,9 @@ const isToken = t => t instanceof Prism.Token;
 const renderPrismTokens = (tokens, styles) => {
   return tokens.map((t, i) => {
     if (isToken(t)) {
+      console.log(t.type);
       return (
-        <span key={`${t.type}-${i}`} className={css(styles[t.type])}>
+        <span key={`${t.type}-${i}`} className={css(styles[t.type])} data-type={`${t.type}`}>
```

Comment:
No. I left it just in case we see an odd highlighting, to know exactly what token is. But I've checked almost all code snippets so there's no reason to keep it.